src/librbd/cache/pwl/LogEntry.cc

```diff
@@ -76,9 +76,11 @@ std::ostream &operator<<(std::ostream &os,
 void WriteLogEntry::init(bool has_data, std::vector<WriteBufferAllocation>::iterator allocation,
                          uint64_t current_sync_gen, uint64_t last_op_sequence_num, bool persist_on_flush) {
   ram_entry.has_data = 1;
+  #ifdef WITH_RBD_RWL
```

Comment:
What's the plan for if you have both modes enabled at build-time? At some point these logs will need to dynamically support either mode of operation.